py_pdf_parser/tests/test_filtering.py

```diff
@@ -157,13 +157,17 @@ def test_filter_by_font(self):
         self.assertEqual(len(doc.elements.filter_by_font("hello,1")), 0)
 
         self.assertEqual(len(doc.elements.filter_by_font("foo,2")), 1)
+        # Check if "foo,2" has been added to cache
+        self.assertEqual(doc._elements_indexes_by_font, {"foo,2": {0}})
```

Comment:
I appreciate there's no styleguide in this project for this, but we don't use the curly bracket notation for sets anywhere in this project so perhaps this should be `set([0])`? (and elsewhere in this file)